.github/CODEOWNERS

```
@@ -3,15 +3,146 @@
+/packages/message-manager/package.json      @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/message-manager/changelog.md      @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+
+## package /packages/name-controller
+/packages/name-controller                   @MetaMask/confirmations
+/packages/name-controller/package.json      @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+/packages/name-controller/changelog.md      @MetaMask/confirmations @MetaMask/wallet-framework-engineers
+
+## package /packages/network-controller
+/packages/network-controller           @MetaMask/wallet-framework-engineers @MetaMask/metamask-assets
+
+## package /packages/notification-controller
+/packages/notification-controller              @MetaMask/notifications
+/packages/notification-controller/package.json @MetaMask/notifications @MetaMask/wallet-framework-engineers
+/packages/notification-controller/changelog.md @MetaMask/notifications @MetaMask/wallet-framework-engineers
+
+## package /packages/notification-services-controller
+/packages/notification-services-controller              @MetaMask/notifications
+/packages/notification-services-controller/package.json @MetaMask/notifications @MetaMask/wallet-framework-engineers
+/packages/notification-services-controller/changelog.md @MetaMask/notifications @MetaMask/wallet-framework-engineers
+
+## package /packages/permission-controller
+/packages/permission-controller        @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers
+
+## package /packages/permission-log-controller
+/packages/permission-log-controller    @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers
+
+## package /packages/phishing-controller
+/packages/phishing-controller              @MetaMask/product-safety
+/packages/phishing-controller/package.json @MetaMask/product-safety @MetaMask/wallet-framework-engineers
+/packages/phishing-controller/changelog.md @MetaMask/product-safety @MetaMask/wallet-framework-engineers
+
+## package /packages/polling-controller
+/packages/polling-controller           @MetaMask/wallet-framework-engineers
+
+## package /packages/preferences-controller
+/packages/preferences-controller       @MetaMask/wallet-framework-engineers
+
+## package /packages/profile-sync-controller
+/packages/profile-sync-controller              @MetaMask/notifications
+/packages/profile-sync-controller/package.json @MetaMask/notifications @MetaMask/wallet-framework-engineers
+/packages/profile-sync-controller/changelog.md @MetaMask/notifications @MetaMask/wallet-framework-engineers
+
+## package /packages/queued-request-controller
+/packages/queued-request-controller               @MetaMask/wallet-api-platform-engineers
+/packages/queued-request-controller/package.json  @MetaMask/wallet-framework-engineers
+/packages/queued-request-controller/changelog.md  @MetaMask/wallet-framework-engineers
```

Is this intentional? Not sure why we'd want to keep api platform team from independently pushing updates to changelogs and manifests in this package.

```suggestion
/packages/queued-request-controller/package.json  @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers
/packages/queued-request-controller/changelog.md  @MetaMask/wallet-api-platform-engineers @MetaMask/wallet-framework-engineers
```